src/frontends/tensorflow/src/op_table.cpp

```diff
@@ -218,6 +218,7 @@ const std::map<std::string, CreatorFunction> get_supported_ops() {
         {"BroadcastTo", CreatorFunction(translate_broadcast_to_op)},
```

Comment:
please also update https://github.com/openvinotoolkit/openvino/blob/master/src/frontends/tensorflow/docs/supported_ops.md. Without this change Ci will fail.

Reply:
sure